### PR TITLE
not the clearest questions

### PR DESCRIPTION
**Pull request by timbo-timboa**
See my changes to the instructions for PROBLEM 8, the last problem, lines ~227.

logCounter isn't coded anywhere but is mentioned in the instructions as if it were. I believe the instructions meant to say "setTimeout"?

Then, the timing of the logs is more accurate in my edit I believe

### CODE REVIEW

File: closures.js [hunk 1]
```diff
@@ -221,13 +240,13 @@ var allPotentialFriends = "?";
...
- 5 seconds after call - log 4
- 6 seconds after call - log 5
- However, because each call to logCounter occurs after the loop has finished,
+ immediatly - log 0
+ 1 seconds after call - log 1
+ 2 seconds after call - log 2
+ 3 seconds after call - log 3
+ 4 seconds after call - log 4
+ 5 seconds after call - log 5
+ However, because each call to setTimeout occurs after the loop has finished,
```
Comment (r-walsh) [inline closures.js:249]: Maybe change it to "each call to console.log occurs after the loop has finished" for more accuracy

**Comment by r-walsh** [comment]
Good catch @timothydm42 , 100% my fault. 

I left a comment on the changes, I think changing from the wording there will be a bit more accurate.

If you address that and remove solution code, I'm happy to merge!

**Comment by r-walsh** [comment]
Thanks for bringing this to us, fixed in #18

**Comment by timbo-timboa** [comment]
thank you for your responsiveness glad I could help. Next time hopefully I
won't be a newb and I'll make a merge-able branch for you

**Comment by r-walsh** [comment]
No worries! Thanks for bringing this to us :+1:

**Pull request closed without merging**